ajax/libs/weui/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "weui",
+  "version": "0.3.0",
+  "description": "为微信 Web 服务量身设计",
```

Comment:
The desc of the repo is better:
`A UI library by WeChat official design team, includes the most useful widgets/modules in mobile web applications.`